### PR TITLE
Release candidate 1.0.0

### PR DESCRIPTION
**Pull request by anastasiia-bilova**

### CODE REVIEW

**Comment by dmytrostriletskyi** [comment]
I can leave first few overall review points before total code review:
- Create guide within documentation (`README`) about environment variable: which type of variables, how to set and why.
- Write message within documentation, why you leave private tokens and urls (for example, it is an unsupported and unavailable data).

File: coverage.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <path fill="#dfb317" d="M63 0h36v20H63z"/>
+        <path fill="url(#b)" d="M0 0h99v20H0z"/>
+    </g>
+    <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
+        <text x="31.5" y="15" fill="#010101" fill-opacity=".3">coverage</text>
+        <text x="31.5" y="14">coverage</text>
+        <text x="80" y="15" fill="#010101" fill-opacity=".3">64%</text>
+        <text x="80" y="14">64%</text>
+    </g>
+</svg>
```
Comment (dmytrostriletskyi) [inline coverage.svg]: Remove this file, it is not necessary for project. It is a result of coverage that you need to ignore (`gitignore`).

File: lyricsbot/domains/songlyrics/songlyrics.py [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
+    from lyricsbot.domains.songlyrics.config import SONGLYRICS_DOWNLOAD_URL
+    from lyricsbot.domains.songlyrics.utils import (
+        make_suitable_url_parameters,
+        remove_punctuation_symbols,
+    )
+
 
-)
+# if the user ask lyrics without 'Press me!' button or enters incorrect data
+LYRICS_WITHOUT_PRESSME_BUTTON = 'Sorry, we have no'
```
Comment (dmytrostriletskyi) [inline lyricsbot/domains/songlyrics/songlyrics.py]: .. we have no what?

File: lyricsbot/errors.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+"""
+Database errors.
+"""
+
+
+class ConnectionError(Exception):  # pylint: disable=W0622
```
Comment (dmytrostriletskyi) [inline lyricsbot/errors.py]: Seems `psycopg2` has own errors. Even if you want custom, name it according to docstring (database connection error).

**Review by dmytrostriletskyi — APPROVED** [review]

File: coverage.svg (hunk 1 above)
Comment (anastasiia-bilova) [reply coverage.svg]: Done.

File: lyricsbot/domains/songlyrics/songlyrics.py (hunk 2 above)
Comment (anastasiia-bilova) [reply lyricsbot/domains/songlyrics/songlyrics.py]: Done.

File: lyricsbot/errors.py (hunk 3 above)
Comment (anastasiia-bilova) [reply lyricsbot/errors.py]: Done.

**Pull request merged**